### PR TITLE
Minor changes in heading

### PR DESCRIPTION
**Pull request by joinimran**
Minor changes in heading as this should go with key trust deployment guide.

Problem: https://github.com/MicrosoftDocs/windows-itpro-docs/issues/5695

### CODE REVIEW

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

**Comment by mypil** [comment]
@mapalko - PR has been copy edited and is ready for final review. This is based on issue #5695

Thank you.

cc: @kenwith

File: windows/security/identity-protection/hello-for-business/hello-key-trust-validate-pki.md [hunk 1]
```diff
@@ -196,7 +196,7 @@ Alternatively, you can forcefully trigger automatic certificate enrollment using
 Use the event logs to monitor certificate enrollment and archive.  Review the configuration, such as publishing certificate templates to issuing certificate authority and the allow auto enrollment permissions.
 
 
-## Follow the Windows Hello for Business on premises certificate trust deployment guide
+## Follow the Windows Hello for Business on premises certificate key trust deployment guide
```
Comment (mapalko) [inline windows/security/identity-protection/hello-for-business/hello-key-trust-validate-pki.md]:
```suggestion
## Follow the Windows Hello for Business on premises key trust deployment guide
```

**Comment by joinimran** [comment]
Thanks, ready to go live.

**Comment by illfated** [comment]
#sign-off

**Comment by mypil** [comment]
@Dansimp - Can you please assist to merge this PR as approved by @mapalko?

Thank you.

**Pull request merged**